prov/rxd/src/rxd_init.c

```diff
@@ -147,6 +153,10 @@ RXD_INI
 			"Maximum number of peers to track (default: 1024)");
 	fi_param_define(&rxd_prov, "max_unacked", FI_PARAM_INT,
 			"Maximum number of packets to send at once (default: 128)");
+	fi_param_define(&rxd_prov, "rescan", FI_PARAM_BOOL,
+			"Enables dynamic network interface detection by rescanning "
+			"system interfaces on each fi_getinfo() invocation. "
+			"(default: -1)");
```

Comment:
I'd modify the text to something like: "Force or disable rescanning for network interface changes... (default: unset)".  Replace -1 with unset, since -1 doesn't make much sense to the user reading this as a boolean value.  Or simply drop the default text completely.  I would make the text clear that setting this to true will force rescanning, while setting it to false will disable rescanning.

Reply:
Thanks @shefty! Here is how the text describing the env var now looks like:
```
fi_param_define(&rxd_prov, "rescan", FI_PARAM_BOOL,
                "Force or disable rescanning for network interface changes. "
                "Setting this to true will force rescanning on each fi_getinfo() invocation; "
                "setting it to false will disable rescanning. (default: unset)");
```

Reply:
thanks -- this was my last concern